indico/MaKaC/services/implementation/search.py

```diff
@@ -22,10 +22,10 @@
 from MaKaC.common.fossilize import fossilize
 
 from indico.modules.groups import GroupProxy
-
-#################################
-# User and group search
-#################################
+from indico.modules.events.models.events import Event
+from indico.modules.events.models.persons import EventPerson
+from indico.modules.events.util import serialize_event_person
+from indico.modules.users.util import _build_match
```

Comment:
This function should be used only within the module since the underscore indicates it is not part of the module public API. However, I would only change the name since (and maybe move it to util module) the function itself is pretty generic and could be used from other places.